src/containers/TexterTodoList.jsx

```diff
@@ -49,23 +50,44 @@ class TexterTodoList extends React.Component {
     }
   }
 
+  textingTurnedOffForOrganization(todos){
+    const orgDataForFirstTodo = todos[0].campaign.organization
+    return (todos[0].campaign.organization && orgDataForFirstTodo.textingTurnedOff ? true : false)
+  }
+
   render() {
     this.termsAgreed()
     const todos = this.props.data.currentUser.todos
     const renderedTodos = this.renderTodoList(todos)
-
+    const textingTurnedOff = this.textingTurnedOffForOrganization(todos)
+    
     const empty = (
       <Empty
         title='You have nothing to do!'
         icon={<Check />}
       />
     )
 
+    const textingOff = (
+      <Empty
```

Comment:
We should maybe make this a container with a route so that we redirect to a page where there's no polling.